cirras-underwriting-api/deploy-tools/server.xml

```diff
@@ -11,14 +11,14 @@
   </GlobalNamingResources>
 
   <Service name="Catalina">
-    <Connector port="#{TOMCAT_PORT}#"
+    <Connector port="${TOMCAT_PORT}"
       protocol="org.apache.coyote.http11.Http11NioProtocol"
-      maxThreads="#{MAX_THREADS}#"
-        connectionTimeout="#{CONNECTION_TIMEOUT}#"
+      maxThreads="${MAX_THREADS}"
```

Comment:
This is called TOMCAT_MAX_THREADS in the config map. Or is that not where it comes from?